### ISSUE TITLE
[Bug] Screen-reader Flag Ignored Unless Started From Library

### ISSUE THREAD

**Issue by arinoch** [binary-knight/usurper-reborn#69]
## Bug Description
The screen-reader flag doesn't work if I launch the game from the start menu. I need to launch it through Steam for it to work as expected. Yes, it's set to be the default in the Steam library.

## Steps to Reproduce
1. Set the --screen-reader flag to be the default in the Steam library
2. From either the start menu or the desktop shortcut, launch Usurper

## Game State
| Property | Value |
|----------|-------|
| Version | 0.52.11 (The Hook) |
| Build Type | Steam |
| Platform | Windows x64 |
| OS | Microsoft Windows NT 10.0.19045.0 |
| Character | Arinoch — Level 1 Human Magician |
| HP | 57/57 |
| Location | MainStreet |
| Play Time | 8m 28s |

<details>
<summary>Recent Debug Log (click to expand)</summary>

```
[19:55:44.174] [INF] [IMMIGRATION] Generated immigrant: Soren Ravenswood (Dwarf Cleric L7 Male)
[19:56:14.182] [INF] [IMMIGRATION] Generated immigrant: Erland Emberglow (Troll Ranger L4 Male)
[19:56:14.186] [INF] [WORLD] Challenge: Leoric Dunmore vs Lysander Silverbrook - Lysander Silverbrook won
[19:56:44.187] [INF] [IMMIGRATION] Generated immigrant: Brandt Ravenswood (Orc Assassin L2 Male)
[19:56:44.189] [INF] [LIFECYCLE] Gwyneth Thornbury is pregnant by Leoric Yarrow (affair=True)! Due: 02:56
[19:57:14.189] [INF] [IMMIGRATION] Generated immigrant: Noric Ravenswood (Gnome Assassin L2 Male)
[19:57:44.191] [INF] [IMMIGRATION] Generated immigrant: Malakai Kettleburn (Gnoll Jester L7 Male)
[19:57:44.194] [INF] [WORLD] Brandt Oakshield stole 28g from Orina Greystone
[19:57:47.845] [INF] [SAVE] Saving 'Arinoch' - Level:1 HP:57/57 Gold:2000
[19:57:47.845] [INF] [GOLD_AUDIT] SAVE: Arinoch Lv1 gold=2,000 bank=0 earned=0
[19:58:14.195] [INF] [IMMIGRATION] Generated immigrant: Vesper Copperfield (Mutant Alchemist L1 Female)
[19:58:14.197] [INF] [WORLD] Orina Greystone stole 28g from Ondine Everhart
[19:58:44.200] [INF] [WORLD] Ondine Everhart stole 13g from Orina Greystone
[19:59:14.201] [INF] [WORLD] Orina Greystone stole 17g from Ondine Everhart
[19:59:14.202] [INF] [WORLD] Orina Greystone stole 17g from Iskander Coldstream
[19:59:44.206] [INF] [WORLD] Orina Greystone stole 22g from Ondine Everhart
[20:00:14.207] [INF] [NPC] Queued Ondine Everhart for respawn (20 ticks = ~10.0 min)
[20:00:30.233] [INF] [GAME] New game started for 'Arinoch'
[20:00:30.236] [INF] [LOCATION] Entered Main Street (ID: MainStreet)
[20:00:44.208] [INF] [IMMIGRATION] Generated immigrant: Ysolde Whitmore (Elf Sage L6 Female)
```
</details>

---
_Reported via in-game bug reporter at 2026-03-17 20:02:01_
